<launch>


  <!-- Avvia il tuo nodo -->
  <node name="gps_odometer" pkg="first_project" type="gps_odometer" output="screen">
    <param name="reference_latitude" type="double" value="45.618932386592405" />
    <param name="reference_longitude" type="double" value="9.281178887031235" />
    <param name="reference_altitude" type="double" value="229.04906147731415" />
  </node>

  <!-- Avvia la bag -->
  <node pkg="rosbag" type="play" name="bag_player" output="screen"
        args="$(find first_project)/bags/project.bag --clock --loop" />
</launch>
